lib/internal/bootstrap/node.js

```diff
@@ -294,36 +294,6 @@ function setupGlobalProxy() {
     enumerable: false,
     configurable: true
   });
-
-  function makeGetter(name) {
-    return deprecate(function() {
-      return this;
-    }, `'${name}' is deprecated, use 'global'`, 'DEP0016');
-  }
-
-  function makeSetter(name) {
-    return deprecate(function(value) {
-      ObjectDefineProperty(this, name, {
-        configurable: true,
-        writable: true,
-        enumerable: true,
-        value: value
-      });
-    }, `'${name}' is deprecated, use 'global'`, 'DEP0016');
-  }
-
-  ObjectDefineProperties(global, {
```

Comment:
With this line removed, the file fails linting because `ObjectDefineProperties` is defined but never used.